charts/talos-cloud-controller-manager/values-example.yaml

```diff
@@ -53,7 +52,8 @@ transformations:
       node-role.kubernetes.io/db: ""
 
 # Deploy the Talos Cloud Controller Manager as a DaemonSet
-useDaemonSet: true
+daemonSet:
+  enabled: true
```

Comment:
Thank you for your PR!!! 

Can we keep the `daemonSet.enable` option set to `false` by default? It's a rare case when you need the daemonSet setup, and it's only necessary with the `nodeIPAM` controller.

Reply:
Sure, I've kept this default for my setup. Will set default to `false`.